src/PanelItem/index.tsx

```diff
@@ -6,52 +6,37 @@ import styles from '../Panel/panel.module.scss'
 interface Props {
   /** Name to pass to the panel for ADA when used with Tabs component */
   name: string
+  /** index is required when using multiple panels with the same name */
+  index?: number
   /** Pre-designated panel item styles */
   type?: 'header' | 'body' | 'footer' | 'link'
   /** Class to pass to the panel item */
-  itemClass?: string
-  /** index & selectedIndex required when using a panel for a tabs component */
-  index?: number
-  /** The currently selected index */
-  selectedIndex?: number
-  /** Set custom inline css */
-  customStyling?: CSSProperties
-  [rest: string]: unknown // ...rest property
+  className?: string
+  /** is the item selected? */
+  isSelected?: number
+  [rest: string]: unknown
 }
 
-const PanelItem: FunctionComponent<Props> = ({
-  name,
-  type,
-  index,
-  selectedIndex,
-  itemClass,
-  customStyling,
-  ...rest
-}) => {
+const PanelItem: FunctionComponent<Props> = ({ name, type, index, isSelected, className, ...rest }) => {
   let Component = 'div'
   if (type == 'link') {
     Component = 'a'
   }
 
-  const isSelected = index === selectedIndex
-
   return (
     <Component
-      id={`panel-${name}-${index}`}
-      // @ts-ignore
+      id={`panel-${name}${index ? `-${index}-` : ''}`}
       className={clsx(
         styles['panel-item'],
         isSelected && styles['active'],
         !isSelected && styles['hidden'],
```

Comment:
Same here ^ Can (or should) this get updated to `styles.hidden`?

Reply:
I'm too lazy to change it